### ISSUE TITLE
Editor spams websocket error messages

### ISSUE THREAD

**Issue by LeXXik** [playcanvas/editor#1213]
If Editor is in the other tab for a while, the connection times out. Once it does, it spams the error messages:

![image](https://github.com/user-attachments/assets/aeeeec31-d65a-4553-9ad4-ce23a8eb0ceb)

**Comment by LeXXik** [comment]
Right, that's just the sending method. Not sure what is calling it, but probably it shouldn't, if the socket state is not right.

**Comment by kpal81xd** [comment]
Yea its to do with a custom override in the socket on close handler that tries to clean up document state before calling the original on close event
